paper-tab.css

```diff
@@ -32,18 +34,7 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 }
 
 #ink {
-  position: absolute;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-  color: #ffff8d;
-}
-
-:host[noink] #ink {
-  pointer-events: none;
-}
-
-:host-context(paper-tabs[noink]) #ink {
+  z-index: 1;
```

Comment:
Is it feasible to re-order the elements in the DOM rather than resort to `z-index`? By moving the `<paper-ripple>` above `<tab-content>` in the DOM, document order will stack them correctly.